Shopstagram - rozwiązanie/css/base/_colors.scss

```diff
@@ -1,6 +1,7 @@
 $colorWhite: #ffffff;
 $colorBlack: #000000;
-$primaryLighter: #fafafa !default;
+// $primaryLighter: #fafafa !default;
+$primaryLighter: rgba(255, 255, 255, 1);
```

Comment:
Tu nie miałam pomysłu jak to zrobić. Czy powinnam stworzyć nową zmienną dla koloru i użyć defaulta?

Reply:
chyba nie do końca rozumiem pytanie i to co chciałas tutaj zrobić, jak coś to pisz na priv, wyjaśnimy temat